packages/documentation-site/patternfly-docs/content/demos/Compass/CompassIntegrationsDemo.tsx

```diff
@@ -0,0 +1,604 @@
+                | "info"
+                | "custom"
+                | undefined
+              }
+              isCompact
+              variant="outline"
+            >
+              {statusText}
+            </Label>
+          ),
+        },
+        {
+          cell: <ActionsColumn items={rowActions} />,
+          props: { isActionCell: true },
+        },
+      ],
+    })
+  );
+
+  const columns = [
+    {
+      cell: undefined,
+      props: {
+        select: {
+          onSelect: () => { },
+          isSelected: false,
+        },
+      },
+    },
+    "Name",
+    "Type",
+    "Status",
+  ];
+
+  const tableViewContent = (
+    <>
+      <DataViewToolbar
+        clearAllFilters={() => { }}
+        filters={
+          <DataViewTextFilter
+            filterId="name"
+            title="Name"
+            placeholder="Filter by name"
+          />
+        }
+        actions={
+          <ToggleGroup>
+            <ToggleGroupItem
+              icon={<ThIcon />}
+              aria-label="grid icon button"
+              isSelected={activeDisplay === "card"}
+              onChange={() => setActiveDisplay("card")}
+            ></ToggleGroupItem>
+            <ToggleGroupItem
+              icon={<ListIcon />}
+              aria-label="list icon button"
+              isSelected={activeDisplay === "table"}
+              onChange={() => setActiveDisplay("table")}
+            ></ToggleGroupItem>
+          </ToggleGroup>
+        }
+        pagination={<Pagination page={1} perPage={10} isCompact />}
+      />
+      <DataViewTable
+        aria-label="Integrations table"
+        columns={columns}
+        rows={rows}
+        className="pf-m-plain"
+      />
+    </>
+  );
+
+  const mainContent = (
+    <>
+      <CompassMainHeader title={<Title headingLevel="h1">Integrations</Title>} toolbar={(
+        <Toolbar hasNoPadding>
+          <ToolbarContent>
+            <ToolbarGroup>
+              <ToolbarItem>
+                <Button
+                  variant="primary"
+                  onClick={() => setActiveDisplay("add")}
+                >
+                  Add integration
+                </Button>
+              </ToolbarItem>
+              <ToolbarItem>
+                <Button variant="secondary">Test integration</Button>
+              </ToolbarItem>
+            </ToolbarGroup>
+            <ToolbarGroup>
+              <ToolbarItem>
+                <Button icon={<EllipsisVIcon />} variant="plain" />
+              </ToolbarItem>
+            </ToolbarGroup>
+          </ToolbarContent>
+        </Toolbar>
+      )} />
+      <CompassContent>
+        <CompassPanel isScrollable>
+          {(() => {
+            if (activeDisplay === "table") {
+              return tableViewContent;
+            } else {
+              return cardViewContent;
+            }
+          })()}
+        </CompassPanel>
+      </CompassContent>
+    </>
+  );
+  const sidebarEndContent = sidebarContent;
```

Comment:
And again for consistencies sake I copied the east and west icon sets from the POC which are different sizes/icon sets.

Also I used the sparkle icon instead of the copy in the middle.

Like with the others I'm good with either approach, just think we should be consistent.